nginx_proxy/start_nginx.sh

```diff
@@ -79,12 +94,44 @@ if [[ -f "${CONF_FILE}" ]]; then
   cp "${CONF_FILE}" /etc/nginx/nginx.conf
 fi
 
-# fetch Service Configuration from Service Management if the service name and
-# service version are provided.
-if [[ -n "${ENDPOINTS_SERVICE_NAME}" && \
-      -n "${ENDPOINTS_SERVICE_VERSION}" ]]; then
-  /usr/sbin/fetch_service_config.sh \
-    -s "${ENDPOINTS_SERVICE_NAME}" -v "${ENDPOINTS_SERVICE_VERSION}" || exit $?
+# If endpoint service name is specified, custom startup script will start
+# the nginx with the specified version or version from rollout information.
+# Otherwise, nginx will be started without the endpoint configuration.
+
+if [[ -z "${ENDPOINTS_SERVICE_NAME}" ]]; then
+  /usr/sbin/nginx -p /usr -c /etc/nginx/nginx.conf
+  exit $?
+fi
+
+if [[ "${ENDPOINTS_ROLLOUT_STRATEGY}"  && \
+      "${ENDPOINTS_ROLLOUT_STRATEGY}" != 'fixed' && \
+      "${ENDPOINTS_ROLLOUT_STRATEGY}" != 'managed' ]]; then
+  echo 'Error: rollout strategy option should be either fixed or managed'
+  usage
+fi
+
+if [[ "${ENDPOINTS_ROLLOUT_STRATEGY}" != 'managed' && \
```

Comment:
Please change this to == 'fixed', that is easier to reason about.

Reply:
Done